lightly/cli/upload_cli.py

```diff
@@ -52,7 +52,7 @@ def _upload_cli(cfg, is_cli_call=True):
         mode = cfg['upload']
         dataset = LightlyDataset(input_dir=input_dir, transform=transform)
         api_workflow_client.upload_dataset(
-            input=dataset, mode=mode
+            input=dataset, mode=mode, max_workers=cfg['loader']['num_workers']
```

Comment:
Careful, we also do concurrent uploads, see [here](https://github.com/lightly-ai/lightly/blob/fc50052f9b2dc62dad9e4823783a471c17c58831/lightly/api/api_workflow_upload_dataset.py#L96). Did you test this?

Reply:
I do not get why to be careful, wasn't it intended to do it like this? Before, always 8 workers were used, see the issue.
Yes I tested it in both ways described in **How to test**

Reply:
Ah yes, my fault. I thought you had added it to the constructor of a `LightlyDataset`.

Reply:
Just safeguard it for `0` as this will break the `ThreadPoolExecutor` and is currently the default argument I believe and then I'll approve it :)

Reply:
I safeguarded it for <1.
The default argument was 1, so this was no problem.